src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -147,9 +147,12 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
         List<FeedbackSessionAttributes> sessions =
```

Comment:
The course existence check above can be avoided if this method receives a CourseAttributes object instead. Check where this method is called and see if the callers can pass the CourseAttributes object instead. If can, do as a separate issue.